howso/direct/client.py

```diff
@@ -737,6 +747,11 @@ def create_trainee(  # noqa: C901
         if features is None:
             features = {}
 
+        if library_type is not None:
+            warnings.warn("create_trainee(library_type=) is deprecated")
```

Comment:
The message should include what to use instead and the category should be `DeprecationWarning`

Reply:
Copied the form of other `DeprecationWarning` in the four places new warnings were added.